src/main/webapp/app/communication/forwarded-message/forwarded-message.component.ts

```diff
@@ -27,13 +27,14 @@ export class ForwardedMessageComponent implements AfterViewInit {
 
     sourceName: string | undefined = '';
     todayFlag?: string;
-    originalPostDetails = input<Posting>();
+    originalPostDetails = input<Posting | null>();
     messageContent = viewChild<ElementRef>('messageContent');
     isContentLong = false;
     showFullForwardedMessage = false;
     postingIsOfToday = false;
 
     protected viewButtonVisible = false;
+    protected hasOriginalPostBeenDeleted = false;
```

Comment:
can you consider using a computed here? For references: https://angular.dev/guide/signals